bearing_seal_element.py

```diff
@@ -0,0 +1,448 @@
+import warnings
+import numpy as np
+import pandas as pd
+import scipy.interpolate as interpolate
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+from ross.data_io.read_xl import (
```

Comment:
Remove unused imports.

Reply:
I did both, removed the IsotSealElement class and the unused import. The commit is [here](https://github.com/rodrigomoliveira1/ross/commit/9e5c5d12ec11d74baab8a443c61f502ee4e0369e)